pom.xml

```diff
@@ -6,7 +6,7 @@
     <parent>
         <groupId>com.boozallen.aissemble</groupId>
         <artifactId>aissemble-parent</artifactId>
-        <version>2</version>
+        <version>3-SNAPSHOT</version>
```

Comment:
I: Can be updated when `aissemble-parent` is released. (See related PR boozallen/aissemble-parent#3)